buddyboss/buddyboss-platform
# Feature/2121 - Activity - Comment - Media/document/video Enhancement

### All Submissions:






### Changes proposed in this Pull Request:

For this I have created hierarchical array and based on that migrate comment.

Fixes #2121.

### How to test the changes in this Pull Request:

1. First you need to add multiple comments with multiple replies with media and documents with latest release branch
2. Add comment above added media and document, and also add multiple replies there ( Check media/document in Photos/Documents Dir ) - https://screencast-o-matic.com/watch/crjXXCVj3jZ
2. After that move this branch and go to the Tools - Repair Community -> Repair Media Comments on the Activity/News Feed -> checked it -> Click on Repair Items
3. Media/Doc comment should not be display in Photos/Documents Dir

### Proof Screenshots or Video

https://screencast-o-matic.com/watch/crjXXvVj31B

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

Dependent API
https://github.com/buddyboss/buddyboss-platform-api/pull/314
Dependent App Native
https://github.com/buddyboss/buddyboss-app-native/issues/1565
